topic_resources/javascript/digital-clock.md

```diff
@@ -0,0 +1,21 @@
+# Digital Clock
+
+### Create a digital clock in your browser!
+
+### Getting Started
+- Create a function which will do all of the work to calculate and show the time. Start by loading this function using the document ready.
+
+  ```
+  $(document).ready( function() {
```

Comment:
Yeah I wasn't sure if we wanted to just throw that one piece in there first, or use this. Either way it's probably a knowledge leap